test/t8_schemes/t8_gtest_face_descendant.cxx

```diff
@@ -92,7 +92,6 @@ class class_descendant: public TestDFS {
   SetUp () override
   {
     dfs_test_setup ();
-    max_test_lvl = scheme->get_maxlevel (eclass);
```

Comment:
Should test level 0 go down to the maxlevel?
So that we have at  least one case where we really test everything?
(I am aware that this will not reduce the runtime for the testlevel 0 and am open to discussion.)

Reply:
We can do it, if it bothers us to much, we can change it later on.